reversed.go

```diff
@@ -0,0 +1,19 @@
+package do
+
+import "reflect"
+
+/*
+Reversed returns the values of a given interface on a reversed basis.
+*/
+func Reversed(iterable interface{}) interface{} {
+	iterValue := getIterable(iterable)
+	returnValue := reflect.MakeSlice(iterValue.Type(), iterValue.Len(), iterValue.Len())
+
+	j := 0
```

Comment:
your right. now less arguing and more hacking (this was done on the fly on my side, you got my drift)